### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader:Functionality is not accessible through keyboard i.e. "TAB" while narrator focus goes on the functionality(Group).

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4204]
Environment Details:
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 70.0.3538.77 (Official Build) (64-bit)
Operating System:Windows10

Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4. Navigate to Code Cell block.

Actual
Functionality is not accessible through keyboard i.e. "TAB" while narrator focus goes on the functionality(Group).

While navigating  through Caps Lock Arrow Keys The narrator reads it the instructions such as "
"click to scroll output; double click to hide""

Expected:
Either the functionality(Group) should be accessible with the help of keyboard or Narrator focus should not go on it and narrator should not announce any information regarding it.
 
User Impact:
Narrator will get confused if information that is not visually present on the page is read out.

Recommendation:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}}

[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip](https://github.com/jupyter/notebook/files/2576116/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
